app/src/main/res/layout/fragment_goal_preview.xml

```diff
@@ -47,7 +47,7 @@ Salt, to taste" />
                 android:onClick="@{() -> handler.onCreateIdeaListClick(viewModel)}"
                 android:src="@drawable/ic_create"
                 android:tint="@color/colorButton"
-                android:backgroundTint="@color/colorAccent"
+                app:backgroundTint="@color/colorAccent"
```

Comment:
Hmm, for some reason that button is green in this page on my phone if I didn't override this color. It seems normal in the rest of the page. nvm, I'll figure it out, thanks for the fix!